client/app/common/user/user.service.js

```diff
@@ -13,42 +10,19 @@ class UserService extends base {
   }
 
   create(user) {
-    if (UserService.validate(["email", "password", "display_name"])) {
-      return this.$http.post("/api/users/", user).then((res) => res.data);
-    } else {
-      return Promise.reject();
-    }
+    return this.$http.post("/api/users/", user).then((res) => res.data);
   }
 
   get(pk) {
-    pk = UserService.resolvePrivateKey(pk);
-    if (!pk)
-      return Promise.reject();
-
-    return this.$http.get(`/api/users/${pk}/`).then((res) => res.data).then((user) => {
-      if (UserService.validate(user)) {
-        return Promise.resolve(user);
-      } else {
-        return Promise.reject();
-      }
-    });
+    return this.$http.get(`/api/users/${pk}/`).then((res) => res.data);
   }
 
-  save(id, updates) {
-    if (UserService.validate(["id"])) {
-      return this.$http.patch(`/api/users/${id}/`, updates).then((res) => res.data);
-    } else {
-      return Promise.reject();
-    }
+  save(user) {
+    return this.$http.patch(`/api/users/${user.id}/`, user).then((res) => res.data);
```

Comment:
All the changes are already merged, as far as i saw it.
So, i will delete the pull request and branch.